site/src/sphinx/server-annotated-service.rst

```diff
@@ -434,6 +438,54 @@ converting ``MyRequest``.
             @RequestObject(MySpecialRequestConverter.class) MyRequest myRequest) { ... }
     }
 
+
+Injecting value of parameters and HTTP headers to a Java object
+"""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+
+BeanRequestConverterFunction_ is a built-in request converter for Java object. You can use it by putting
+`@RequestObject`_ annotation on the parameters which you want to convert.
+
+.. code-block:: java
+
+    public class MyAnnotatedServiceClass {
+        @Post("/hello")
+        public HttpResponse hello(@RequestObject MyRequestObject myRequestObject) { ... }
+    }
+
+Beside this, you also need to inject value of path parameters, HTTP parameters, or HTTP headers to the Java
+object, by putting `@Param`_ or `@Header`_ annotations on the following elements of the Java object:
+
+- fields
+- or, constructors with only one parameter
+- or, methods with only one parameter
+- or, parameters of constructors
+- or, parameters of methods
+
+
+.. code-block:: java
+
+    public class MyRequestObject {
+        @Param("name") // this field will be injected by value of parameter "name"
```

Comment:
- Please start a sentence with a capitalized word, e.g. this -> This
- Please end a sentence with a period.